crates/fuel-core/src/state/rocks_db.rs

```diff
@@ -863,6 +867,33 @@ fn next_prefix(mut prefix: Vec<u8>) -> Option<Vec<u8>> {
     None
 }
 
+impl<Description> KeyValueMutate for RocksDb<Description>
```

Comment:
I wonder if this should be behind a feature flag, since it wasn't exposed publicly before.

Reply:
it should be behind the `rocksdb` feature flag, and it doesn't affect any other api's :)

Reply:
It should be behind the "test-helpers" feature. We shouldn't allow modification of the Rocksdb outside of the `commit` method

Reply:
addressed in #2159